polylogue/cli/click_app.py

```
@@ -0,0 +1,619 @@
+
+# ---------------------------- misc ----------------------------
+
+
+@cli.command()
+@click.argument("provider", required=False)
+@click.option("--command", "cmd", type=str, help="Filter by command (e.g., sync codex)")
+@click.option("--print", "print_only", is_flag=True, help="Only print the path without opening")
+@click.option("--json", is_flag=True, help="Emit JSON with the last run info")
+@click.option("--fallback", type=click.Path(path_type=Path), help="Fallback path if no run is found")
+@click.pass_obj
+def open(env: CommandEnv, **kwargs) -> None:  # type: ignore[func-returns-value]
```

Ruff (0.15.9)

[error] Variable `open` is shadowing a Python builtin

(A001)